datarockets-style.gemspec

```diff
@@ -32,10 +32,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 1.56.3"
+  spec.add_dependency "rubocop", "~> 1.65.1"
   spec.add_dependency "rubocop-graphql", "~> 1.4"
   spec.add_dependency "rubocop-rails", "~> 2.21"
```

Comment:
And this to [2.26](https://github.com/rubocop/rubocop-rails/releases/tag/v2.26.0)?